css/index.css

```diff
@@ -1,15 +1,30 @@
-.row {
-width: 100%;
-background-color: #FB9300;
+
+.container{
+    display: flex;
+    justify-content: space-evenly;
+    background-color: backdrop wheat;
 }
 
 .cards{ 
-width: 50px;
-height: 100px;
-border: solid 5px #F54748; 
-/* INNER TEXT #LOREM */
+    width: 250px;
+    height: 500px;
+    border: solid 5px #F54748; 
```

Comment:
Similar to above, try to use one of our color palette selections for the border color!